l10n_br_account_banking_payment/models/account_payment.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today - KMEE (<http://kmee.com.br>).
+#  Luis Felipe Miléo - mileo@kmee.com.br
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.addons import decimal_precision as dp
+from odoo.tools.float_utils import float_round as round
+from odoo.exceptions import ValidationError
+
+
+class PaymentLine(models.Model):
+    _inherit = 'account.payment.line'
+
+    @api.model
+    def _get_info_partner(self, partner_record):
```

Comment:
If you insist in adding this if, you should do the next:

```suggestion
    def _get_info_partner(self, partner_record=False):
```